google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,159 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.debugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.debugger.v2
+  go:
+    package_name: cloud.google.com/go/devtools/clouddebugger/apiv2
+  csharp:
+    package_name: Google.Devtools.Clouddebugger.V2
+  ruby:
+    package_name: Google::Cloud::Devtools::Clouddebugger::V2
+  php:
+    package_name: Google\Cloud\Devtools\Clouddebugger\V2
+  nodejs:
+    package_name: '@google-cloud/clouddebugger'
+interfaces:
+- name: google.devtools.clouddebugger.v2.Debugger2
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 60000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000
+    total_timeout_millis: 600000
+  methods:
+  - name: SetBreakpoint
+    flattening:
+      groups:
+      - parameters:
+        - debuggee_id
+        - breakpoint
+        - client_version
+    required_fields:
+    - debuggee_id
+    - breakpoint
+    - client_version
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: GetBreakpoint
+    flattening:
+      groups:
+      - parameters:
+        - debuggee_id
+        - breakpoint_id
+        - client_version
+    required_fields:
+    - debuggee_id
+    - breakpoint_id
+    - client_version
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: DeleteBreakpoint
+    flattening:
+      groups:
+      - parameters:
+        - debuggee_id
+        - breakpoint_id
+        - client_version
+    required_fields:
+    - debuggee_id
+    - breakpoint_id
+    - client_version
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: ListBreakpoints
+    required_fields:
```

Comment:
This didn't generate a flattening section because the request has more than 4 fields. Only two of them are required, though: Should we apply the flattening threshold only to the number of required fields?